docs/doxygen/migration/shadow.txt

```diff
@@ -0,0 +1,15 @@
+/**
+@page shadow_migration Shadow
+@brief How to migrate a Shadow application from v3 to 202009.00.
+
+The Device Shadow library has been redesigned in this release.  The new design has the following differences than the v3 of the AWS IoT Device SDK for Embedded C.
+
+- The new Device Shadow library is completely decoupled from an underlying MQTT library. The v3 library depends on the same version of the MQTT library in the AWS IoT Device SDK for Embedded C. <br>
+- The new library does not have any code for handling JSON files. The v3 library contained code for assembling simple JSON files in the format expected by the Device Shadow service. <br>
```

Comment:
```suggestion
- The new Device Shadow library does not have any code for constructing JSON files. The v3 Device Shadow library, on the other hand, contained code for constructing JSON files in the format expected by the AWS IoT Device Shadow Service. <br>
```

Reply:
plz check [the commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1226/commits/f504c65a1d8030ae4237312b7b63b563b8700ca8).